components/brave_new_tab_ui/async/brave_vpn.ts

```diff
@@ -42,7 +43,7 @@ handler.on(Actions.launchVPNPanel.getType(), async (store) => {
   getNTPBrowserAPI().pageHandler.reportVPNWidgetUsage()
 })
 
-handler.on<string>(
+handler.on<BraveVPN.ManageURLType>(
   Actions.openVPNAccountPage.getType(),
   async (store, payload) => {
     getNTPBrowserAPI().pageHandler.openVPNAccountPage(payload)
```

Comment:
This gives `Property 'openVPNAccountPage' does not exist on type 'PageHandlerRemote'` on linux because
`brave_new_tab_page.mojom` is updated like below and it's not enabled on linux.
@fallaciousreasoning Do you think it's possible that execute these methond only when they exists?
```
  [EnableIf=enable_brave_vpn]
  RefreshVPNState();

  [EnableIf=enable_brave_vpn]
  LaunchVPNPanel();
  OpenVPNAccountPage(string intent);

  [EnableIf=enable_brave_vpn]
  OpenVPNAccountPage(brave_vpn.mojom.ManageURLType type);

  [EnableIf=enable_brave_vpn]
  ReportVPNWidgetUsage();
```

Reply:
If we know we aren't going to call it on Linux, you could add a `// ts-expect-error` to ignore the errors, otherwise yeah, you might need to just have it always show up.

Reply:
If it was in Chromium webui we could use the preprocess_if_expr stuff, but we don't support that :/

Reply:
yeah, I also searched about preprocess here.
Added `@ts-expect-error`. presubmit check complains when I use `@ts-ignore`.
Thanks for the tip! 👍🏼

Reply:
I think we're in a weird situation..

When using `@ts-ignore`, presubmit complains (`TS2578: Unused '@ts-expect-error' directive).`

When using `@ts-expect-error`,  build failed with `Unused '@ts-expect-error' directive.`

Reply:
I guess maybe a @ts-ignore with a pragma for presubmit too?

Reply:
Suppressed by `// eslint-disable-next-line @typescript-eslint/prefer-ts-expect-error`. Hope it works :)